structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,125 @@
+
+## What Community Collab isn’t
+
+- It isn’t an official affiliation with the Quilt project itself or any other community part of the program
+- It isn’t a place to blindly synchronise bans between communities
+    - Each community must decide on appropriate moderation action based on facts shown to them and their own rules and policies. Infringing this rule may get your community removed from the Community Collab program.
+
+## Roles inside the server
+
+### Keyholder
+
+The keyholder for this space is the same as for the Quilt Community Team. This role is described in the [RFC 0007 # Keyholders](https://github.com/QuiltMC/rfcs/blob/master/structure/0007-community-team.md#keyholders).
+
+### Managers
+
+Managers are responsible for keeping the collab program up and running.
+
+- Maintain and organise the associated Discord server
+- Actioning changes to the collab-related spaces
+- Organise polls and votes as required by formal processes
+- Help resolve conflicts between communities
+- Stay in touch with any community part of the program
+
+Community Collab Managers are the same as Community Managers of the Quilt Community Team.
+
+### Collaborators
+
+Collaborators are moderators of one or more communities in the program. They have access to the server to the moderation part of the server, such as general moderator chats and case threads. Access is also given to the event category.
+
+### Event Planners
+
+Event Planners are members of the staff team of one or more communities in the program, who focuses on event organisation. They have access to the event part of the server.
+
+### Project Roles
+
+Other roles may be given surrounding the different projects happening inside the program. Those roles are informal and will depend on each project.
+
+### Invitees
+
+Invitees are temporary users, with no access to the rest of the server. They only have access to temporary channels required by the process they are visiting for.
+
+## Voting process
+
+If required, a vote among the Community Collab group can be called by a Manager. A poll and an associated thread are created in an internal channel designated for that purpose, including as many details as possible.
+
+Each community gets a single vote representing its member. They are free to write their own process for determining their vote, but it must follow those rules:
+
+- This process must be democratic and give a voice to any community member who is able to join the Community Collab server under the Collaborator role.
+- Each eligible voter must have single, equal voting power.
+- The process can remove the voting power under some inactivity threshold.
+
+Once a community has decided their vote, one member can report their vote in the thread, mentioning the community name, and the option being voted for, with corresponding proof (such as a screenshot of an internal poll).
```

Comment:
I feel as if this is asking too much of a lot of communities. If I had to guess, very few servers actually have any sort of formal voting system - Quilt is the first place I've been which actually has one.

Additionally, some communities are considerably more active in CC than others, and there are some that I haven't seen participating at all. What happens if simply nobody from the community responds, ever? I don't think it would be reasonable for us to go after them for this,

Reply:
If a community doesn't report anything, they simply aren't counted in the poll, which is fine. I would be okay with a message such as "does everyone agree to X" with a few upvotes on them. We at least need something for transparency.